### PR TITLE
commit an example print statement with my name

### PR DESCRIPTION
**Pull request by shivammpat**

### CODE REVIEW

**Review by carltonwu — APPROVED** [review]
that is a pretty cool name!

File: training.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Hi! My name is Shivam Patel.")
```
Comment (WilliamFreitag) [inline training.py:1]: The formal greeting is Hello. Not "Hi." Are you not in an academic setting?

**Review by WilliamFreitag — CHANGES_REQUESTED** [review]

File: training.py (hunk 1 above)
Comment (carltonwu) [reply training.py:1]: based

**Review by carltonwu — APPROVED** [review]

**Review by WilliamFreitag — APPROVED** [review]
Nicely done!

**Pull request merged**